src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1662,6 +1664,36 @@ class spell_dru_wild_growth_aura : public AuraScript
     }
 };
 
+// 145108 - Ysera's Gift
+class spell_dru_yseras_gift : public AuraScript
+{
+    PrepareAuraScript(spell_dru_yseras_gift);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo
+        ({
+            SPELL_DRUID_YSERAS_GIFT_HEAL_SELF,
+            SPELL_DRUID_YSERAS_GIFT_HEAL_PARTY
+        });
+    }
+
+    void HandleEffectPeriodic(AuraEffect const* aurEff)
+    {
+        int32 healAmount = int32(GetTarget()->CountPctFromMaxHealth(aurEff->GetAmount()));
```

Comment:
there are effects that increase the % health amount, is there a way to pass target as in CalcValue() so that such effects are taken into account?

Reply:
thats already taken care of in `AuraEffect::CalculateAmount`

Reply:
no, we should not, aurEff amount is already modified